### PR TITLE
feat(args): printing multiple errors in p4, submit

### PR DESCRIPTION
**Pull request by i-keliukh**
Also removed tests with deleting fields from settings objects. If the
fields is absent, the error state mechanism is able to detect it and
remember the error. However, the code in the module still can access the
field. Such accesses result in exceptions raised from gravity.

This is not an issue for normal operation, because when user launches
the script, all settings fields are present.

### CODE REVIEW

**Comment by i-keliukh** [comment]
Relates to #261

File: universum/modules/error_state.py [hunk 1]
```diff
@@ -21,10 +21,14 @@ class HasErrorState(Module):
...
+        self.global_error_state: GlobalErrorState = self.global_error_state_factory()
 
     def error(self, message: str):
         self.global_error_state.errors.append(inspect.cleandoc(message))
 
-    def is_in_error_state(self):
+    def is_in_error_state(self) -> bool:
         return self.global_error_state.is_in_error_state()
+
+    def check_required_option(self, setting_name, message) -> None:
```
Comment (o-andrieiev) [inline universum/modules/error_state.py:32]: why did you omit types in setting_name and message?

Comment (i-keliukh) [reply universum/modules/error_state.py:32]: Is there some mode for mypy to warn about this?

Comment (o-andrieiev) [reply universum/modules/error_state.py]: --disallow-incomplete-defs

File: universum/modules/error_state.py [hunk 2]
```diff
@@ -21,10 +21,14 @@ class HasErrorState(Module):
 
     def __init__(self, *args, **kwargs) -> None:
         super().__init__(*args, **kwargs)  # type: ignore
-        self.global_error_state = self.global_error_state_factory()
+        self.global_error_state: GlobalErrorState = self.global_error_state_factory()
 
     def error(self, message: str):
```
Comment (o-andrieiev) [inline universum/modules/error_state.py:26]: -> None missing

**Review by k-dovgan — APPROVED** [review]

**Review by k-dovgan — APPROVED** [review]

**Pull request merged**